src/AnalyticsEngine/App.ControlPanel.Engine/SqlExtentions/Profiling-03-CreateSchema.sql

```diff
@@ -1310,7 +1450,14 @@ BEGIN
 
         [Yammer Posted] BIGINT DEFAULT 0,
         [Yammer Read] BIGINT DEFAULT 0,
-        [Yammer Liked] BIGINT DEFAULT 0
+        [Yammer Liked] BIGINT DEFAULT 0,
+
+
+        -- Add Copilot fields here
+        -- TODO
+        [Copilot ChatsCount] BIGINT DEFAULT 0,
+        [Copilot MeetingsCount] BIGINT DEFAULT 0,
+        [Copilot FilesCount] BIGINT DEFAULT 0
       );
```

Comment:
Please use separate words for the metric names. Makes them friendlier and they don't need to be changed later in the report.